.github/workflows/bump-datadog-ci.yml

```diff
@@ -1,7 +1,11 @@
-# This workflow automatically creates a release PR for the CI integration.
+# This workflow automatically creates a PR bumping datadog-ci in the CI integration.
 
 name: Bump Datadog CI
 
+env:
+  GIT_AUTHOR_EMAIL: 'packages@datadoghq.com'
+  GIT_AUTHOR_NAME: 'ci.datadog-ci'
```

Comment:
This means that the initiator of this automation system as a whole is the CI of https://github.com/DataDog/datadog-ci.

I used the same convention as in [datadog-api-client-typescript](https://github.com/DataDog/datadog-api-client-typescript/blob/670d6274f4d8fad885efcf4b095cd775114ff5a3/.github/workflows/prepare_release.yml#L5), where the author name is `ci.datadog-api-spec` because the initiator is the CI of https://github.com/DataDog/datadog-api-spec.